### DISCUSSION TITLE
Correlate same source with different setups in different regions

### DISCUSSION THREAD

**Discussion by Tomoya-Iizawa** [scikit-hep/cabinetry#461] [Q&A]
Hi,

I would like to correlate same source with different setups in different regions, is it possible? I mean, for example, I would like to include Ztautau-PP8 uncertainty with shape for 1b region while only normalization for 2b region. I tried to implement defining the two with just same name, but "all systematic names must be unique" error happens and it does not work.

Best,
Tomoya

**Comment by alexander-held** [comment]
Hi Tomoya, you can correlate two entries in the list of systematic uncertainties via the `ModifierName` setting, see https://cabinetry.readthedocs.io/en/latest/advanced.html#manually-correlating-systematics.
